requirements.txt

```diff
@@ -13,8 +13,9 @@ colorama==0.4.1
 pymongo==4.6.3
 backoff==1.10.0
 scanpy==1.9.2
-anndata==0.9.1
+anndata==0.8.0
```

Comment:
We've been on anndata 0.9.1 for over a year, is there a particular need for 0.8.0? 
anndata is actually up to 0.11 (also, I don't _think_ we have a specific dependency on anndata version, we may not need to enumerate anndata in requirements.txt <- how would one go about confirming before implementing such a change?). It probably only makes sense to roll back if CAS has a specific need but I'd hope we can figure out a way to move towards a more recent release of the library.

Reply:
Good question.  When I installed CAS, it needed this version of `anndata`.